src/main/java/bio/terra/service/notification/NotificationService.java

```diff
@@ -0,0 +1,58 @@
+package bio.terra.service.notification;
+
+import bio.terra.app.configuration.NotificationConfiguration;
+import bio.terra.common.iam.AuthenticatedUserRequest;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.common.annotations.VisibleForTesting;
+import jakarta.annotation.PostConstruct;
+import java.io.IOException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Service;
+
+@Service
+public class NotificationService {
+
+  private static final Logger logger = LoggerFactory.getLogger(NotificationService.class);
+
+  private final PubSubService pubSubService;
+  private final NotificationConfiguration notificationConfiguration;
+  private final ObjectMapper objectMapper;
+
+  public NotificationService(
+      PubSubService pubSubService,
+      NotificationConfiguration notificationConfiguration,
+      ObjectMapper objectMapper) {
+    this.pubSubService = pubSubService;
+    this.notificationConfiguration = notificationConfiguration;
+    this.objectMapper = objectMapper;
+  }
+
+  @VisibleForTesting
+  @PostConstruct
+  protected void createTopic() {
+    try {
+      pubSubService.createTopic(
+          notificationConfiguration.projectId(), notificationConfiguration.topicId());
+    } catch (IOException e) {
+      logger.error("Error creating notification topic", e);
```

Comment:
Since we expect to see this in production, what would you think about reducing the log level here? Maybe to `warn` or `info`? And how would you feel about including a note that it is expected in prod? Maybe including some details about why we are doing this?